### DISCUSSION TITLE
API SDK clients c++?

### DISCUSSION THREAD

**Discussion by ejrua** [pocketbase/pocketbase#617] [Q&A]
API SDK clients c++, in future?

**Comment by ganigeorgiev** [comment]
At least until v1.0.0, I'm not planning to work on other SDKs. Community contributions are welcomed (please note that some breaking changes are expected in relation to #376).

You could also check this discussion - https://github.com/pocketbase/pocketbase/discussions/391.

**Reply by ejrua** [reply]
thank
